robottelo/cli/base.py

```diff
@@ -374,6 +374,13 @@ def _construct_command(cls, options=None):
                 if val is True:
```

Comment:
The indentation level in this method gets pretty nutty. Can we try to rein it in a little bit by doing the following:

```python
for key, val in options.items():
    if val is None:
        continue
    if val is True:
        ...
```